react/README.md

```diff
@@ -291,7 +291,7 @@
   )}
 
   // good
-  {todos.map((todo) =>
+  {todos.map(todo =>
```

Comment:
thanks! this example is actually also violating another rule, that multiline jsx must be wrapped with parens, ie `=> (` and `))}` - could you make that fix and amend this commit and force push?